contrib/job-scripts/coreudmp-handling/post.d/001-restore-core-pattern

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh -e
```

Comment:
Would you please include the license and short description of what this script does?